webview/src/plots/components/App.test.tsx

```diff
@@ -49,6 +49,7 @@ import {
   toggleDragAndDropMode as toggleCustomPlotsDragAndDropMode
```

Comment:
Next steps after this would be updating section title tooltips to include an error. Full design ideas list is in https://github.com/iterative/vscode-dvc/issues/5087#issuecomment-1859295790